lib/astarte_core/cql_utils.ex

```diff
@@ -167,4 +167,40 @@ defmodule Astarte.Core.CQLUtils do
 
     eid
   end
+
+  @spec get_from_realm!(any()) :: any()
+  def get_from_realm!("") do
+    raise "EmptyRealm"
+  end
+
+  def get_from_realm!(realm_name) when is_binary(realm_name) do
+    customer_keyspace = :persistent_term.get("instance_id")
```

Comment:
This should be either retrieved via config (see e.g. [Astarte Data Access envs](https://github.com/astarte-platform/astarte_data_access/blob/master/lib/astarte_data_access/config.ex)) or passed as an argument to the function

Reply:
actually, direct access of persistent_term or whatever env value from main application managed by Skogsra could not work, so this value needs to be passed as an argument